puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -16,6 +16,15 @@
           ensure => 'present',
           before => Class['postgresql::globals'],
         }
+        yumrepo { 'foreman-infra-el7':
+          baseurl =>  'https://yum.theforeman.org/infra/el7/',
+          enabled => true,
+          require => Class['postgresql::globals'],
+        }
+        package { 'rh-postgresql12-postgresql-evr':
+          ensure  => 'present',
+          require =>  yumrepo['foreman-infra-el7'],
+        }
```

Comment:
The requires there is good, but I think `postgresql::globals` won't ensure the package is installed. I am thinking this needs to happen after https://github.com/theforeman/foreman-infra/pull/1285/files#diff-8fa48bcc6aed22cd7a1aa8b436d72823R87

Reply:
That makes sense.  And you're referring to the package requiring `postgresql::server::role`?

Reply:
Actually, you can probably just do:

```
requires => Package['postgresql::globals::server_package_name']
```